src/models/profile.ts

```diff
@@ -117,6 +118,14 @@ export const Profile = t.intersection([
     // https://it.wikipedia.org/wiki/ISO_3166-2
     preferredLanguages: PreferredLanguages,
 
+    // This parameter specifies how a specific user wants to visualize push notifications.
+    // FULL leads to descriptive push notifications while ANONYMOUS leads to silent ones.
+    // (defaults to UNSET)
+    pushNotificationsContentType: withDefault(
+      t.union([PushNotificationsContentType, t.literal("UNSET")]),
+      "UNSET"
```

Comment:
What if we define a global `UNSET` type for this scenario?
```typescrypt
export const UNSET = t.literal("UNSET");
export type UNSET = t.TypeOf<typeof UNSET>;
export const UNSET_VALUE: UNSET = "UNSET";
```

```suggestion
      t.union([PushNotificationsContentType, UNSET]),
      UNSET_VALUE
```

Reply:
done in https://github.com/pagopa/io-functions-commons/pull/297/commits/25f802f12c34ec047393044f870ae527936bc452